api/RTCIceTransport.json

```diff
@@ -5,43 +5,43 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCIceTransport",
         "support": {
           "webview_android": {
-            "version_added": null
+            "version_added": false
           },
           "chrome": {
-            "version_added": null
+            "version_added": false
           },
           "chrome_android": {
-            "version_added": null
+            "version_added": false
           },
           "edge": {
-            "version_added": null
+            "version_added": "15"
           },
           "edge_mobile": {
-            "version_added": null
+            "version_added": false
```

Comment:
Out of curiosity, how did you determine that it's not supported in Edge Mobile, given that it is supported in Edge?

Reply:
That would be because I accidentally changed this when I didn't mean to. :)